benchmark/streamly-benchmarks.cabal

```diff
@@ -544,3 +544,13 @@ benchmark Unicode.Utf8
   type: exitcode-stdio-1.0
   hs-source-dirs: Streamly/Benchmark/Unicode
   main-is: Utf8.hs
+
+-------------------------------------------------------------------------------
+-- Window Benchmarks
+-------------------------------------------------------------------------------
+
+benchmark Window
```

Comment:
The benchmark should be named as `Data.Fold.Window` .
Move it next to `Data.Fold`.